clappr/clappr-ios
# Create Container & Playback layers

## 🏁 Goal
Continue the implementation of the visual layers abstraction by adding Container and Playback layers.

## ✅ How to test
1. Add the following snippet to LayerComposer.swift#13
```swift
playbackLayer.backgroundColor = .yellow
containerLayer.backgroundColor = .red
```
2. Run the Clappr_Example project
3. Enable view hierarchy debugger
4. You should be able to see the new layers

## 🖼 Screenshots
![Screen Shot 2020-08-28 at 14 10 59](https://user-images.githubusercontent.com/2274568/91594500-6af36d80-e938-11ea-8c06-5c989334a562.png)
